### DISCUSSION TITLE
Pausing and generating successive answers/trains of thoughts

### DISCUSSION THREAD

**Discussion by Fusseldieb** [open-webui/open-webui#5392] [General]
On the official ChatGPT interface, when it outputs stateful Python notebooks and now chains of thoughts, there's a way for the model to "finish" outputting content, process for a bit while showing some sort of progress, and then resuming the answer.

Is this possible with Pipelines?

In other words, how would I structure the pipeline so it can pause for a brief moment, process, and then resume the answer? Is this even possible currently?

**Comment by tjbck** [comment]
This should be doable using Filters, would love to see a community contribution here!

**Reply by Fusseldieb** [reply]
How would I start doing something like this? Aren't filters only for preprocessing?
